src/hid/hid-device.cpp

```diff
@@ -384,5 +430,12 @@ namespace librealsense
 
             return ep;
         }
+
+        void rs_hid_device::set_gyro_scale_factor(double scale_factor) 
+        {
+            _gyro_scale_factor = scale_factor;
+            if( scale_factor == 10000.0 )
```

Comment:
Please add a comment about this if, why 10000?

Reply:
Done